components/serverless/hack/webhook_config.yaml

```diff
@@ -1,78 +0,0 @@
-reservedEnvs:
```

Comment:
Why this is removed? How it's connected with PR subject?

Reply:
There is no more webhook and I noticed the old file